package.json

```diff
@@ -20,8 +20,10 @@
     "test": "node_modules/.bin/mocha",
     "lint-staged": "node_modules/.bin/lint-staged",
     "coverage-report": "node --loader ts-node/esm util/gen_coverage_report.js",
-    "find-translations": "node --loader ts-node/esm util/find_missing_translations.js",
-    "translate-timeline": "node --loader ts-node/esm util/translate_timeline.ts",
+    "util": "node --loader ts-node/esm util/index.ts",
+    "find-translations": "npm run util -- findTranslations",
+    "translate-timeline": "npm run util -- translateTimeline",
```

Comment:
### bug 1 (file selection error)

```
$ npm run translate-timeline

> cactbot@ translate-timeline C:\Users\enne\cactbot
> npm run util -- translateTimeline


> cactbot@ util C:\Users\enne\cactbot
> node --loader ts-node/esm util/index.ts "translateTimeline"

(node:29084) ExperimentalWarning: --experimental-loader is an experimental feature. T
his feature could change at any time
(Use `node --trace-warnings ...` to show where the warning was created)
? Input a valid timeline filename:  ui\raidboss\data\02-arr\raid\t11.txt
? Select a locale:  fr
Couldn't find 'ui\raidboss\data\02-arr\raid\t11.txt', aborting.
```

### bug 2 (no output)

This command works before this patch, but not with this patch.

```
$ npm run translate-timeline -- -l fr -t t11

> cactbot@ translate-timeline C:\Users\enne\cactbot
> npm run util -- translateTimeline "-l" "fr" "-t" "t11"


> cactbot@ util C:\Users\enne\cactbot
> node --loader ts-node/esm util/index.ts "translateTimeline" "-l" "fr" "-t" "t11"

(node:1596) ExperimentalWarning: --experimental-loader is an experimental feature. Th
is feature could change at any time
(Use `node --trace-warnings ...` to show where the warning was created)
```

Reply:
Bug 2 fixed, that is because I forgot to fetch arguments from `args` when it has values.
But I can't reproduce Bug 1..

Reply:
Bug 2 is fixed for me.  Bug 1 still happens.

I don't know why Bug 1 is not happening for you.  It looks like `findTriggersFile` gets `ui\raidboss\data\02-arr\raid\t11.txt` as a parameter, and it will never find that as a filename.  How does this function work for you?

Reply:
I don't know, either. It runs properly on my side.

I can see a difference that I run it on a Linux machine, and you run on Windows?

<details><summary>Images (Click to Expand)</summary>

![Snipaste_2021-07-19_08-33-26](https://user-images.githubusercontent.com/19927330/126087317-5e3ecfc5-8140-47ee-a1ac-3cc7dcc00437.png)
![Snipaste_2021-07-19_08-33-51](https://user-images.githubusercontent.com/19927330/126087318-28f98c1e-7488-49f3-bf7b-34125fc251e4.png)

</details>

Reply:
```diff
-  shortName = shortName.replace(/\.(?:[jt]s|txt)$/, '');
+  shortName = shortName.replace(/\.(?:[jt]s|txt)$/, '').split(path.sep).join(path.posix.sep);
```
Fixes this for me and appears to be backwards compatible with the old script invocation for both Git Bash and Command Prompt from Windows.

I'm assuming this is a no-op for Linux given what the code does, but still might be worth a quick test.

Reply:
Looks like the code you wrote would replace all the path separators to the POSIX one (`/`)? I have no idea, but why is it compatible with Windows?

Reply:
Thanks, that works for me on Windows!